### DISCUSSION TITLE
extend native names to star names, asterisms and dso names in sky cultures

### DISCUSSION THREAD

**Discussion by kajaji** [Stellarium/stellarium#1964] [Ideas]
In defining a sky culture, native names can be provided for constellations and planets. Can this be extended to star, asterism and DSO names?
This will make the display consistent when selecting "native" for sky culture.

**Comment by gzotti** [comment]
Yes. We have a bunch of ideas around the skycultures. Better naming, translation, labelling is among them. Please be patient, though, this is a major project. I would not want to add another quick patch now.

**Reply by kajaji** [reply]
Thanks.
